config.json

```diff
@@ -11,404 +11,404 @@
       ]
     },
     {
-      "slug": "accumulate",
+      "slug": "sum-of-multiples",
       "difficulty": 1,
       "topics": [
       ]
     },
     {
-      "slug": "sublist",
```

Comment:
I think it was too hard to be at the start, because people don't have yet a good understanding of recursing.